packages/material-ui/src/locale/index.js

```diff
@@ -811,14 +875,18 @@ export const isIS = {
 };
 
 export const itIT = {
+  localization: {
+    formatNumber: formatNumber('it-IT'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@Angelk90 Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).